crates/common/tedge_utils/src/file.rs

```diff
@@ -133,6 +136,10 @@ impl PermissionEntry {
         {
             Ok(mut f) => {
                 self.apply(file)?;
+                f.sync_all().map_err(|from| FileError::FailedToSync {
```

Comment:
Why sync is needed now?

Reply:
to make sure files are written to disk. It's needed for the notify tests